lib/opentok.js

```diff
@@ -995,6 +995,106 @@ OpenTok = function (apiKey, apiSecret, env) {
   this.listCallbacks = callbacks.listCallbacks.bind(null, apiConfig);
 };
 
+/**
+   * Force mute a specific stream in a session with a callback.
+   *
+   * @param sessionId {String} The session ID corresponding to the session where muting a stream.
+   *
+   * @param streamId {String} The stream ID corresponding to the stream to be muted.
+   *
+   * @param callback {Function} The function to call upon completing the operation. One argument
+   * is passed to the function:
+   *
+   * <ul>
+   *
+   *   <li>
+   *      <code>error</code> &mdash; An error object (if the call to the method fails). This is
+   *      set to null if there is no error.
+   *   </li>
+   *
+   * </ul>
+   *
+   * @method #forceMuteStream
+   * @memberof OpenTok
+   */
+
+this.forceMuteStream = function forceMuteStream(sessionId, streamId, callback) {
+  if (!sessionId || typeof sessionId !== 'string') {
+    return callback(new Error('Invalid arguments -- must provide a sessionId string.'));
+  }
+  if (!!streamId && typeof streamId !== 'string') {
+    return callback(new Error('Invalid arguments -- streamId must be a string.'));
+  }
+  if (!callback || typeof callback !== 'function') {
+    return callback(new Error('Invalid arguments -- must provide a callback function.'));
+  }
+  return this.client.forceMute({
+    sessionId: sessionId,
+    streamId: streamId
+  }, function cb(err) {
+    if (err) { return callback(err); }
+    return callback(null);
+  });
+};
+
+/**
+   * Force mute a specific stream in a session with a callback.
+   *
+   * @param sessionId The session ID corresponding to the session where muting a stream.
+   *
+   * @param opts The stream ID corresponding to the stream to be muted.
+   * <ul>
+   *
+   *   <li>
+   *     <code>excudedStreamIds</code> &mdash; An array of Stream IDs to exlude from mute.
+   *   </li>
+   *
+   * </ul>
```

Comment:
This `options` object should have two properties:

* `excludedStreamIds` (Array) -- An array of stream IDs to be excluded from being muted.
* `active` (Boolean) -- Whether streams published after call to the method will have audio muted (`true`) or not (`false`).

Reply:
@jeffswartz